openedx/core/djangoapps/content_libraries/tasks.py

```diff
@@ -127,6 +127,16 @@ def send_events_after_publish(publish_log_pk: int, library_key_str: str) -> None
         elif hasattr(record.entity, "container"):
             container_key = api.library_container_locator(library_key, record.entity.container)
             affected_containers.add(container_key)
+
+            try:
+                # We do need to notify listeners that the parent container(s) have changed,
+                # e.g. so the search index can update the "has_unpublished_changes"
+                for parent_container in api.get_containers_contains_item(container_key):
+                    affected_containers.add(parent_container.container_key)
+                    # TODO: should this be a CONTAINER_CHILD_PUBLISHED event instead of CONTAINER_PUBLISHED ?
```

Comment:
Since we are not doing anything special which would warrant a separate CONTAINER_CHILD_PUBLISHED signal, it should not be required. We are just sending signal for all affected containers.